packages/beacon-node/test/e2e/sync/unknownBlockSync.test.ts

```diff
@@ -47,7 +47,7 @@ describe("sync / unknown block sync", function () {
       this.timeout("10 min");
 
       // the node needs time to transpile/initialize bls worker threads
-      const genesisSlotsDelay = 16;
+      const genesisSlotsDelay = 7;
```

Comment:
why the number 7?

Reply:
It's the one epoch, which seem reasonable to me based on observation of performance.